dvc/remote/s3.py

```diff
@@ -50,13 +50,9 @@ def __init__(self, repo, config):
         url = config.get(Config.SECTION_REMOTE_URL, storagepath)
         self.path_info = self.path_cls(url)
 
-        self.region = os.environ.get("AWS_DEFAULT_REGION") or config.get(
-            Config.SECTION_AWS_REGION
-        )
+        self.region = config.get(Config.SECTION_AWS_REGION)
```

Comment:
Doesn't removing `os.environ.get` prevents us from taking "global" config? Shouldn't here be just a change of order in `or`?

Reply:
@pared what is "global" config? boto is able to read `AWS_` env vars on itsown.

Reply:
@pared  , if I remember correctly, the reason to parse the environment variable was to give them precedence over the configuration.